dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,13 +413,37 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+    val filter: ScopeFilter = ScopeFilter(inProjects(projects: _*))
+    // check if application.conf is present in service
+    val serviceResourceDirectories = ((unmanagedResourceDirectories in Runtime) ?? Nil).all(filter)
+    //Verifying the presence of the required application.conf file
+    val verifyApplicationConfTask = Def.task {
+
+      val servicesPaths: Seq[File] = serviceResourceDirectories.value.flatten
+
+      val (validProject, notValidProject) = servicesPaths.partition(f => {
+        val r: PathFinder = {
+          f.get * "application.conf"
+        }
+        r.get.nonEmpty
+      })
+      //displays all projects that not contain a application.conf file
+      if (notValidProject.nonEmpty) {
+        notValidProject foreach (f => {
+          val log = state.value.log
+          log.error("File application.conf has been missed in the following service >>> " + f.project)
```

Comment:
Thanks for your answer @dotta (and your comprehension :) )

I am agree with you that's better to keep track of the project instead of show paths. I will use your tip to implement it.
Regarding the documentation about application.conf, I could write it in frame of resolution of this issue.
Ok I understand the point about the key :play.modules.enabled. 
Sur, I am agree also that we must know the behavior when the application.con is missing.

I propose in waiting that  @jroper gives his agreement on the solution, I continue to work on the issue #61 and I will finish this issue after. What do you think?